wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift

```diff
@@ -28,7 +28,11 @@ final class ZClientViewController: UIViewController {
     let userSession: UserSession
 
     private(set) var conversationRootViewController: UIViewController?
-    private(set) var currentConversation: ZMConversation?
+    private(set) var currentConversation: ZMConversation? {
+        didSet {
+            DeveloperToolsViewModel.context.currentConversation = currentConversation
+        }
+    }
```

Comment:
This isn't the best place to set the `DeveloperToolsViewModel.context.currentConversation` because it doesn't get set back to `nil` when we navigate out of the conversation. 
My original intention was to set the current conversation when the conversation view controller appears, and reset it when it disappears. But somehow the view controller got deallocated right after being presented, which resulted in the current conversation being reset 😕 
I didn't have time to find a good solution so I ended up with the one above, but it would be better to fix it so that the current conversation is reset properly.

Reply:
I changed it and moved to the `ConversationContentViewController`. `DeveloperToolsViewModel.context.currentConversation` gets `nil` in deinit() method.